<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LC_DuplicateOptions</class>
 <widget class="QWidget" name="LC_DuplicateOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="0" column="4">
    <widget class="QLineEdit" name="leOffsetY">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Vertical offset of duplicate from original entity</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="leOffsetX">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Horizontal offset of duplicate from original entity</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Pen:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="cbInPlace">
     <property name="toolTip">
      <string>If checked, duplicate will be positioned in original's coordinates</string>
     </property>
     <property name="text">
      <string>In Place</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Layer:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QComboBox" name="cbPen">
     <property name="toolTip">
      <string>Pen to apply to created duplicate</string>
     </property>
     <item>
      <property name="text">
       <string comment="pen">Active</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="pen">Original</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="pen">Original Resolved</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QComboBox" name="cbLayer">
     <property name="toolTip">
      <string>Layer to which duplicate should be placed</string>
     </property>
     <item>
      <property name="text">
       <string comment="layer">Active</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="layer">Original</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Offset Y:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Offset X:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
